scipy/sparse/_csr.py

```diff
@@ -75,6 +75,10 @@ class csr_array(_cs_matrix):
       - slow column slicing operations (consider CSC)
       - changes to the sparsity structure are expensive (consider LIL or DOK)
 
+    Canonical Format
+        - In the canonical format, CSR matrices have indices sorted per row,
+          and cannot contain duplicate entries.
```

Comment:
```suggestion
        - Indices are sorted by row.
        - There are no duplicate entries.
```

Reply:
How about:

```suggestion
        - Within each column, indices are sorted by row.
        - There are no duplicate entries.
```

Reply:
For CSR, it needs to be "Within each row, indices are sorted by column."